include/Row.hpp

```diff
@@ -227,11 +227,33 @@ class Row {
     }
 
     /*
+     * Retrieve the value at the specified column index as a geographical point.
+     * This data type is not currently supported by the C++ client so this always
+     * throws an exception.
+     * @throws UnsupportedTypeException always
+     * @return never returns
+     */
+    void getGeographyPoint(int32_t column) {
+        throw UnsupportedTypeException(wireTypeToString(WIRE_TYPE_GEOGRAPHY_POINT));
+    }
+
+    /*
+     * Retrieve the value at the specified column index as a geographical object.
+     * This data type is not currently supported by the C++ client so this always
+     * throws an exception.
+     * @throws UnsupportedTypeException always
+     * @return never returns
+     */
+    void getGeography(int32_t column) {
+        throw UnsupportedTypeException(wireTypeToString(WIRE_TYPE_GEOGRAPHY));
+    }
+
+    /*
      * Returns true if the value at the specified column index is NULL and false otherwise
      * @throws InvalidColumnException The name of the column was invalid.
      * @return true if the value is NULL and false otherwise
      */
-    bool isNull(int32_t column) throw(voltdb::InvalidColumnException) {
+    bool isNull(int32_t column) throw(voltdb::InvalidColumnException, voltdb::UnsupportedTypeException) {
```

Comment:
This is not a terribly serious comment, but throw declarations like these are on their way to being deprecated.  They are not really supported very well in any compiler.  The only one which is supported well is "throw()" which fails if any exception is thrown.  And that is being replaced by a new keyword "noexcept" in C++17.

I wouldn't change this, though.  It might be interesting to see if our compilers honored this.